src/java/org/apache/cassandra/service/accord/txn/TxnNamedRead.java

```diff
@@ -129,6 +129,8 @@ public AsyncChain<Data> read(Timestamp executeAt)
         // this simply looks like the transaction witnessed TTL'd data and the data then expired
         // immediately after the transaction executed, and this simplifies things a great deal
         int nowInSeconds = (int) TimeUnit.MICROSECONDS.toSeconds(executeAt.hlc());
+        if (consistencyLevel == null || consistencyLevel == ConsistencyLevel.ONE)
```

Comment:
`NODE_LOCAL` isn't possible here?

Reply:
It's not supposed to be? `NODE_LOCAL` can't run on Accord, it's not a distributed consistency level.

Reply:
`ONE`, `LOCAL_ONE`, `NODE_LOCAL` would all be cases where strict filtering should happen, but it looks like only `ONE` is in `SUPPORTED_READ_CONSISTENCY_LEVELS`. I think we're good.

Reply:
Hmm will we want to run `NODE_LOCAL` at some point on Accord? Feels like someone enables that later and then this breaks.

Reply:
In `StatementRestrictions` we just do this:

```
// If there is only one replica, we don't need reconciliation at any consistency level.
boolean needsReconciliation = !table.isVirtual()
                              && options.getConsistency().needsReconciliation()
                              && Keyspace.open(table.keyspace).getReplicationStrategy().getReplicationFactor().allReplicas > 1;
```
Maybe we could future-proof by just using the `options.getConsistency().needsReconciliation()` bit?